src/test/java/com/zendesk/maxwell/MaxwellTestJSON.java

```diff
@@ -23,11 +24,43 @@
 		return mapper.readValue(json, MAP_STRING_OBJECT_REF);
 	}
 
+	// If objects aren't equal, and `expected` is of the form /{regex}/, do a pattern match.
+	private static boolean jsonValueMatches(Object value, Object expected) {
+		if ( (value == null && expected != null) || (value != null && expected == null) )
+			return false;
+		if ( value.equals(expected) )
+			return true;
+		if ( !expected.toString().startsWith("/") )
+			return false;
+		String regex = expected.toString().replaceAll("/", "");
+		return Pattern.matches(regex, value.toString());
+	}
+
 	public static void assertJSON(List<Map<String, Object>> jsonOutput, List<Map<String, Object>> jsonAsserts) {
 		ArrayList<Map<String, Object>> missing = new ArrayList<>();
+		Boolean contains;
+
+		for ( Map<String, Object> m : jsonAsserts ) {
+			contains = false;
+
+			for ( Map jm : jsonOutput ) {
+				contains = true;
```

Comment:
I'm not checking if the generated output has more keys than the expected json. Lmk if it is a good idea to do it.